### ISSUE TITLE
Travis CI is failing

### ISSUE THREAD

**Issue by MatonAnthony** [mdn/doc-linter-webextension#4]
**Current behavior:**  Travis CI is erroring due to "browser being undefined"
https://travis-ci.org/mdn/doc-linter-webextension

**Expected behavior:** 
Travis CI should run without hassle.

**Blockers:**
The bug is known and has been filled on bugzilla : https://bugzilla.mozilla.org/show_bug.cgi?id=1316441

**References:**
https://bugzilla.mozilla.org/show_bug.cgi?id=1316441

**Comment by MatonAnthony** [comment]
Travis CI error with "browser being undefined" had been addressed in f55f991d1e9b0af1a1ba1f9a5cd8dff41a6db6d9

There is however an other error:
```log
START:
07 06 2017 15:52:17.962:INFO [karma]: Karma v1.7.0 server started at http://0.0.0.0:9876/
07 06 2017 15:52:17.964:INFO [launcher]: Launching browser Firefox with unlimited concurrency
07 06 2017 15:52:17.971:INFO [launcher]: Starting browser Firefox
07 06 2017 15:52:21.158:INFO [Firefox 55.0.0 (Linux 0.0.0)]: Connected on socket FRqZyK6wva9vXsM0AAAA with id 15979648
Firefox 55.0.0 (Linux 0.0.0) ERROR
  TypeError: port is undefined
  at add-on/worker.js:9
Finished in 0.492 secs / 0 secs @ 15:52:21 GMT+0000 (UTC)
```

**Comment by Elchi3** [comment]
This is fixed now \o/
